### PR TITLE
Database

### PR DESCRIPTION
**Pull request by JoseLopez235**
Pr contains:

removes all user-related user files
adds item files
creates database
adds items through seeds in databse

### CODE REVIEW

File: api/database/models.py [hunk 1]
```diff
@@ -3,34 +3,48 @@
...
+        
+        if gender is not None:
+            gender = bleach.clean(gender).strip()
+            if gender == '':
+                gender = None
+
+        self.name = name
+        self.category = category
+        self.weather = weather
+        self.gender = gender
```
Comment (Eternal-Flame085) [inline api/database/models.py:47]: nice!

File: manage.py [hunk 2]
```diff
@@ -24,11 +24,30 @@ def db_seed():
...
+    db.session.add(item1)
+    db.session.add(item2)
+    db.session.add(item3)
+    db.session.add(item4)
+    db.session.add(item5)
+    db.session.add(item6)
+    db.session.add(item7)
+    db.session.add(item8)
+    db.session.add(item9)
+    db.session.add(item10)
```
Comment (Eternal-Flame085) [inline manage.py:47]: will be annoying to add all of the items manually

**Review by Eternal-Flame085 — APPROVED** [review]

**Review by kevxo — APPROVED** [review]
Awesome, Looks good

File: api/database/models.py [hunk 3]
```diff
@@ -3,34 +3,48 @@
...
+    # category
+    category = Column(String(100), nullable=False)
+    # weather
+    weather = Column(String(80), nullable=False)
+    # gender
+    gender = Column(String(80), nullable=False)
+
+    def __init__(self, name, category, weather, gender):
+        if name is not None:
+            name = bleach.clean(name).strip()
```
Comment (stillsheryl) [inline api/database/models.py:25]: It's great that we're cleaning the data before adding it to the database.

File: api/resources/items.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import bleach
+from flask import request
+from flask_restful import Resource, abort
+from sqlalchemy.orm.exc import NoResultFound
+
+from api import db
+from api.database.models import Item
+
+
+# def _user_payload(user):
```
Comment (stillsheryl) [inline api/resources/items.py:13]: Nice to keep as a reference and we can delete it once we have a better idea of what we're doing. :)

File: manage.py [hunk 5]
```diff
@@ -24,11 +24,30 @@ def db_seed():
     db.create_all()
 
     # seed anything here we might need
-    user = User(username='iandouglas', email='ian.douglas@iandouglas.com')
-    db.session.add(user)
+    item1 = Item(name='beard trimmer', category='toiletries', weather='all', gender='male')
```
Comment (stillsheryl) [inline manage.py:27]: Seeds! This will make testing a lot easier until we can import the full set of data

**Review by stillsheryl — APPROVED** [review]
Great job! Excited with the progress so far!

**Pull request merged**